shuup/admin/modules/permission_groups/__init__.py

```diff
@@ -40,7 +40,7 @@ def get_model_url(self, object, kind, shop=None):
         return derive_model_url(PermissionGroup, "shuup_admin:permission_group", object, kind)
 
     def get_extra_permissions(self) -> Iterable[str]:
-        return ["permission_group.object_selector"]
+        return ["group.object_selector"]
```

Comment:
is this correct?

Reply:
Yes, PermissionGroup comes from: 
`from django.contrib.auth.models import Group as PermissionGroup`

Reply:
ah, makes sense